avogadro/rendering/solid_first_fs.glsl

```diff
@@ -21,14 +21,29 @@ varying vec2 UV;
 
 // RGB rendered texture
 uniform sampler2D inRGBTex;
+
+// RGB color for applying fog
+uniform float fogR;
+uniform float fogG;
+uniform float fogB;
+uniform float offset;
+
 // Depth rendered texture
 uniform sampler2D inDepthTex;
 // 1.0 if enabled, 0.0 if disabled
 uniform float inAoEnabled;
+// 1.0 if enabled, 0.0 if disabled
+uniform float inFogEnabled;
```

Comment:
I know we've done this for inAoEnabled - shouldn't we just do this based on the inFogStrength == 0.0. The C++ code can just set inFogStrength to 0.0 when passing to the shader. Seems like we could do this throughout.